instrumentation/resources/library/src/main/java/io/opentelemetry/instrumentation/resources/CgroupV2ContainerIdExtractor.java

```diff
@@ -25,7 +24,7 @@ class CgroupV2ContainerIdExtractor {
 
   static final Path V2_CGROUP_PATH = Paths.get("/proc/self/mountinfo");
   private static final Pattern CONTAINER_ID_RE = Pattern.compile("^[0-9a-f]{64}$");
-
+  private static final Pattern CONTAINER_ID_RE1 = Pattern.compile("cri-containerd:[0-9a-f]{64}");
```

Comment:
perhaps `CRI_CONTAINER_ID_RE` would be better?
also I think could keep the blank line that separates static and non-static variables

Reply:
@laurit Not all releases of CRI-O has this implementation. Do you think the name `CRI_CONTAINER_ID_RE ` is better and not confusing? Then I will make change.

Reply:
Just changed according to your comment.

Reply:
> Not all releases of CRI-O has this implementation. Do you think the name CRI_CONTAINER_ID_RE  is better and not confusing? 

you are welcome to come up with a better name, I'm sure there is a better name than `CONTAINER_ID_RE1`